boards/arm/sama5/giant-board/include/board.h

```diff
@@ -0,0 +1,195 @@
+#define LED_IRQSENABLED   0  /* Interrupts enabled         OFF       */
+#define LED_STACKCREATED  1  /* Idle stack created         ON        */
+#define LED_INIRQ         2  /* In an interrupt            N/C       */
+#define LED_SIGNAL        2  /* In a signal handler        N/C       */
+#define LED_ASSERTION     2  /* An assertion failed        N/C       */
+#define LED_PANIC         3  /* The system has crashed     Flash     */
+#undef  LED_IDLE             /* MCU is is sleep mode       Not used  */
+
+/* Thus if the Orange LED is statically on, NuttX has successfully  booted
+ * and is, apparently, running normally.
+ * If LED is flashing at approximately 2Hz, then a fatal error has been
+ * detected and the system has halted.
+ */
+
+/* Pin disambiguation *******************************************************/
+
+/* Alternative pin selections are provided with a numeric suffix like _1, _2,
+ * etc. Drivers, however, will use the pin selection without the numeric
+ * suffix.
+ * Additional definitions are required in this board.h file.
+ * For example, if we wanted the PCK0on PB26, then the following definition
+ * should appear in the board.h header file for that board:
+ *
+ *   #define PIO_PMC_PCK0 PIO_PMC_PCK0_1
+ *
+ * The PCK logic will then automatically configure PB26 as the PCK0 pin.
+ */
+
+/* UART1. There is a TTL serial connection available on
+ * pins RX and TX.  This is be driven by UART1.
+ *
+ *   ---- ------------------------ -------------
+ *   J1/2  SCHEMATIC                   SAMA5D2
+ *   PIN   NAME(s)                  PIO  FUNCTION
+ *   ---- ------------------------ -------------
+ *    14   UART1_RX                 PD3  UTXD1
+ *    15   UART1_TX                 PD2  URXD1
+ *   ---- ------------------------ -------------
+ */
+
+#define PIO_UART1_RXD     PIO_UART1_RXD_1
+#define PIO_UART1_TXD     PIO_UART1_TXD_1
+
+/* FLEXCOM4 is available as a UART.
+ *
+ *   ----- ------- -------------
+ *   J1/2  BOARD      SAMA5D2
+ *   PIN   NAME    PIO  FUNCTION
+ *   ----- ------- -------------
+ *    6    AD2     PD21 FLEXCOM4
+ *    9    AD3     PD22 FLEXCOM4
+ *   ----- ------- -------------
+ */
+
+#define PIO_FLEXCOM4_IO0  PIO_FLEXCOM4_IO0_2
+#define PIO_FLEXCOM4_IO1  PIO_FLEXCOM4_IO1_2
+
+/* PIO pins */
+
+#define PIO_TX      IO_
+
+/* SDIO - Used for both Port 0 & 1 ******************************************/
+
+/* 386 KHz for initial inquiry stuff */
+
+#define BOARD_SDMMC_IDMODE_PRESCALER    SDMMC_SYSCTL_SDCLKFS_DIV256
+#define BOARD_SDMMC_IDMODE_DIVISOR      SDMMC_SYSCTL_DVS_DIV(2)
+
+/* 24.8MHz for other modes */
+
+#define BOARD_SDMMC_MMCMODE_PRESCALER   SDMMC_SYSCTL_SDCLKFS_DIV8
+#define BOARD_SDMMC_MMCMODE_DIVISOR     SDMMC_SYSCTL_DVS_DIV(1)
+
+#define BOARD_SDMMC_SD1MODE_PRESCALER   SDMMC_SYSCTL_SDCLKFS_DIV8
+#define BOARD_SDMMC_SD1MODE_DIVISOR     SDMMC_SYSCTL_DVS_DIV(1)
+
+#define BOARD_SDMMC_SD4MODE_PRESCALER   SDMMC_SYSCTL_SDCLKFS_DIV8
+#define BOARD_SDMMC_SD4MODE_DIVISOR     SDMMC_SYSCTL_DVS_DIV(1)
+
```

Comment:
SDIO is STM32 terminology for HSMCI/SDMMC.  These should be removed since SDMMC is not supported by the Giant Board.